rollup.config.js

```diff
@@ -11,6 +11,8 @@ const mode = process.env.NODE_ENV;
 const dev = mode === 'development';
 const legacy = !!process.env.SAPPER_LEGACY_BUILD;
 
+const onwarn = (warning, onwarn) => warning.code === 'CIRCULAR_DEPENDENCY' && warning.message.includes() || onwarn(warning);
```

Comment:
Is that `includes` call supposed to take an argument?

Also: I never bothered internalizing these operator precedences, are you down with putting the `&&` expression inside of parentheses?

Reply:
It is supposed to take an argument, I'm not sure what happened to it. This was the dumb typo I alluded to in chat 😛 

Yeah I can put the `&&` in parentheses.